src/templates/doc-jsx.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
-import { getItemByPath } from 'gatsby-theme-iterative-docs/src/utils/shared/sidebar'
+import { getItemByPath } from '@dvcorg/gatsby-theme-iterative/src/utils/shared/sidebar'
 
-import SEO from 'gatsby-theme-iterative-docs/src/components/SEO'
+import SEO from '@dvcorg/gatsby-theme-iterative/src/components/SEO'
 
-import Documentation from 'gatsby-theme-iterative-docs/src/components/Documentation/WithJSX'
+import Documentation from '@dvcorg/gatsby-theme-iterative/src/components/Documentation/WithJSX'
 
```

Comment:
External links icons aren't the right icon:

<img width="834" alt="image" src="https://user-images.githubusercontent.com/43496356/160925672-826d5e1c-06b0-4588-875f-68b50d4b72ea.png">

Reply:
Good catch, I had to shuffle around some icons to make them importable from the theme and this icon was one of the ones changed most often.

Reply:
[Fixed!](https://dvc-org-try-external-th-doe8zu.herokuapp.com/doc/user-guide/running-dvc-on-windows)
![image](https://user-images.githubusercontent.com/9111807/162116553-820f7c89-0f89-4822-b55f-5bb4e87f30a1.png)


The files in question are `external-link-icon.svg` (the big gray one here) and `external-link.svg` (the correct small blue one). Better names might be in order here, but aside from that I checked for uses of `external-link.svg` in master and found this is the only place that icon is used, and I'm pretty sure we only have two, so this fix should take care of this kind of problem.